packages/core/src/components/forms/_controls.scss

```diff
@@ -169,7 +174,7 @@ $control-checked-background-color-active: nth(map-get($button-intents, "primary"
   </label>
 
   :checked  - Checked
-  :disabled - Disabled
+  :disabled - Disabled. This should be used with a <code>.pt-disabled</code> modifier (not shown below).
```

Comment:
this doesn't explain _how_ to use it. say something about "add .pt-disabled to .pt-control element to change text color," and for all the similar notes below.